code/modules/mob/mommi_modules.dm

```diff
@@ -0,0 +1,89 @@
+
+/obj/item/weapon/robot_module/mommi
```

Comment:
am i having a stroke or is this an exact copypaste from code/modules/mob/living/silicon/mommi/mommi_modules.dm?

Reply:
Yes, it is

Reply:
pfew got worried there for a sec

also good on @boy2mantwicethefam for not noticing it in his review
ya doofus

Reply:
>is this line from code/modules/mob/mommi_modules.dm  the same as code/modules/mob/living/silicon/mommi/mommi_modules.dm?
Are you sure you arent having a stroke? What is issue?

Reply:
The issue is that why do we need this new file? What is its purpose? It's not even ticked in vgstation.dme from what I can tell.

Reply:
I dont understand, I just added my line to the existing mommi_modules.dm, I didnt create a new file?

Reply:
i ams retard

Reply:
like how does that even happen